woocommerce/woocommerce-ios
# Create Copy email action if the Email Message composer is not available

Closes #1442 

Allow the user to copy the email if the email message composer is not available.

**Notes:** I chose to display an action sheet because that's how the other actions are presented for the email & phone buttons on this screen.

### To test
0. Ensure the testing Order has a shipping email address.
1. Open in a simulator or have a device with no Mail accounts set up.
2. Navigate to Order > Order Details > View Billing Details and tap on the Contact Details email address cell.

<img src="https://user-images.githubusercontent.com/1062444/70545899-f0798800-1b33-11ea-9489-cd6b05ad67a4.png" width="350" />

Update release notes:
